### ISSUE TITLE
To load .env at build in the devcontainer.json

### ISSUE THREAD

**Issue by PONzu-0529** [microsoft/vscode-remote-release#5324]
This question is a continuation of the question at the following URL: https://github.com/microsoft/vscode-dev-containers/issues/934

I'm using Docker Desktop in Windows.
Docker Desktop version 3.4.0 or later, seamlessly move to Docker Compose V2.

I tried to use `env_file: .env` in the compose case.
But, when run `docker-compose`, it did not load `.env`.

However, in the container, `.env` is loaded.
Maybe, to set `env_file: .env` in `docker-compose.yml`, `.env` is loaded only after the container is up.

So, I would like to be able to load `.env` at build in the `devcontainer.json`.

Test files is at the following URL: https://github.com/PONzu-0529/test-load-env
In Docker Compose V2, container name ( `container_name: "${ENV}"` ) is null,
but `echo $ENV` in container, return `child`.

**Comment by eitsupi** [comment]
This may be a duplicate of #4885.

**Comment by PONzu-0529** [comment]
@eitsupi 
Thank you for your advice.
Should I comment on an issue if these are duplicates?
( Sorry for the beginner's question. )

**Comment by eitsupi** [comment]
@PONzu-0529 If you think this is a duplicate, it would make sense to close this issue and comment your opinion in #4885.

**Comment by PONzu-0529** [comment]
@eitsupi 
OK!
I will recomment the issue.
Thank you!!!
